### PR TITLE
fix typos in README.md

### PR DESCRIPTION
**Pull request by kungfoome**

### CODE REVIEW

**Review by jeff-mccoy — APPROVED** [review]

**Comment by jeff-mccoy** [comment]
Thanks @kungfoome, we'll keep the same process as [Zarf](https://github.com/defenseunicorns/zarf/blob/main/CONTRIBUTING.md) and have 2 approvers for external PRs. @bdw617, pending your approval.

**Review by bdw617 — APPROVED** [review]
Thank you!

**Pull request merged**
